### ISSUE TITLE
[Question]: Is it posstible to read the tables created using TSQL as  Postgres tables as well?

### ISSUE THREAD

**Issue by abner** [babelfish-for-postgresql/babelfish_extensions#15]
### What's the question?

[Question]: Is it posstible to read the tables created using TSQL in babelfish as  Postgres tables, using psql or another
Postgres clients?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nasbyj** [comment]
Yes, you can, but we haven't really documented how the objects map yet. I've created [Issue 85](https://github.com/babelfish-for-postgresql/babelfish_project_website/issues/85) to track that.

**Comment by JimFinnerty** [comment]
hi @abner .  Yes, to a good first approximation, objects created through the TSQL dialect can be queried from the postgreSQL dialect, but there are differences between the two languages that are important to understand and account for.  TSQL applications generally use a case-insensitive, accent-sensitive database collation and tsql uses this collation to match identifiers case-insensitively (delimited or not), whereas PostgreSQL doesn't support case-insensitive collations at the database level as of PG13, and matches delimited identifiers case-sensitively. Objects created in the TSQL dialect will store identifers in lower case (all column names, for example), and may be truncated and uniquified if the name exceeds the maximum identifier length in PostgreSQL.  The names of the columns as stored in either pg_attribute or the information_schema can be used to show you how the column names are spelled and can be referenced from the postgres dialect.  You can, for example, configure a database administration GUI that can connect to either PostgreSQL or SQL Server to explore how the objects are represented and to submit queries in either dialect.

**Comment by abner** [comment]
Thanks for the feedback. I really don't know why the first time I created the database into the installed postgres with babelfish enabled (which I installed locally here using Docker) I couldn't see the tables created using the PGAdmin with the user.

I think after I followed the new instructions added in this commit https://github.com/babelfish-for-postgresql/babelfish_extensions/commit/210c32bfc61367c6670de552b6ec12e17c32b4c0 I could have
access to the tables in the Postgres side.
